sim/paladin/item_sets_pve_phase_7.go

```diff
@@ -107,6 +107,10 @@ func (paladin *Paladin) applyNaxxramasRetribution6PBonus() {
 				critChanceBonusPct += paladin.GetStat(stats.MeleeCrit)
 			}
 
+			if spell.Matches(ClassSpellMask_PaladinExorcism) {
+				critChanceBonusPct += 100
+			}
```

Comment:
Forgot this last time but the retadins think that the exorcism guaranteed crit against undead counts. I'm skeptical, but we'll see